daemon/container_unix.go

```diff
@@ -1007,13 +1013,15 @@ func (container *Container) connectToNetwork(idOrName string, updateSettings boo
 		}
 	}()
 
-	if err := container.updateEndpointNetworkSettings(n, ep); err != nil {
+	if err = container.updateEndpointNetworkSettings(n, ep); err != nil {
```

Comment:
@aboch for the sake of consistency with `daemon/container.go`, I think it would be good to follow @LK4D4's suggestion.